addons/account/models/account_move.py

```diff
@@ -105,6 +108,7 @@ def _get_default_invoice_incoterm(self):
             ('cancel', 'Cancelled')
         ], string='Status', required=True, readonly=True, copy=False, tracking=True,
         default='draft')
+    posted_before = fields.Boolean(help="Technical field for knowing if the move has been posted before", copy=False)
```

Comment:
Because a draft move that is the first one in its tuple (journal, period) has a name but has not been posted.
(Also because in the previous version, one could have put the name back to '/' and it could have bypassed constraints)

I also personally find it much clearer for someone not familiar with the code to understand than `name != '/'` but I guess that's accessory.